notebooks/Model_Training_with_TensorArray.ipynb

```diff
@@ -0,0 +1,5826 @@
+       "0      [-9.85053703e-02 -4.05019075e-01  7.42887914e-0...             0   \n",
+       "1      [-5.70215769e-02 -4.81121272e-01  9.89868402e-0...             0   \n",
+       "2      [-4.82420959e-02 -2.53300011e-01  1.16719234e+0...             0   \n",
+       "3      [-2.66830117e-01 -3.10087562e-01  1.00747287e+0...             0   \n",
+       "4      [-0.22296937 -0.2130852   0.93310183 -0.2613601...             0   \n",
+       "...                                                   ...           ...   \n",
+       "281104 [ 7.55636692e-01 -9.18912172e-01 -1.40303954e-0...             3   \n",
+       "281105 [-1.15284957e-01 -4.44919914e-01  4.71556604e-0...             0   \n",
+       "281106 [ 4.56021696e-01 -8.97085190e-01  6.78616092e-0...             3   \n",
+       "281107 [-1.97137520e-01 -5.42719424e-01  2.94020921e-0...             0   \n",
+       "281108 [-5.76507807e-01 -4.21606839e-01  9.94703531e-0...             0   \n",
+       "\n",
+       "       predicted_class predicted_iob predicted_type  \\\n",
+       "0                    O             O           None   \n",
+       "1                    O             O           None   \n",
+       "2                    O             O           None   \n",
+       "3                    O             O           None   \n",
+       "4                    O             O           None   \n",
+       "...                ...           ...            ...   \n",
+       "281104           B-ORG             B            ORG   \n",
+       "281105               O             O           None   \n",
+       "281106           B-ORG             B            ORG   \n",
+       "281107               O             O           None   \n",
+       "281108               O             O           None   \n",
+       "\n",
+       "                                       predicted_class_pr  \n",
+       "0      [9.99162793e-01 7.28614070e-06 3.68106932e-04 1...  \n",
+       "1      [9.98624017e-01 1.17727804e-06 2.30890326e-05 6...  \n",
+       "2      [9.99822159e-01 2.92875927e-06 4.63864656e-05 9...  \n",
+       "3      [9.99746151e-01 7.32364501e-09 1.02976297e-05 1...  \n",
+       "4      [9.99962561e-01 3.31572805e-10 3.60350074e-07 4...  \n",
+       "...                                                   ...  \n",
+       "281104 [3.28689827e-07 1.31715175e-09 5.09700063e-09 9...  \n",
+       "281105 [9.99998076e-01 6.02579816e-11 1.43360960e-10 5...  \n",
+       "281106 [1.80262724e-07 4.99876003e-10 2.15961101e-08 9...  \n",
+       "281107 [9.99999401e-01 2.85864619e-11 4.76410155e-11 3...  \n",
+       "281108 [9.99996300e-01 1.40033531e-11 2.56121148e-12 4...  \n",
+       "\n",
+       "[281109 rows x 17 columns]"
+      ]
+     },
+     "execution_count": 39,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "train_results_df = predict_on_df(train_df, int_to_label, base_model)\n",
```

Comment:
I don't think it makes much sense to run inference on the training set. I recommend you skip ahead to running inference on the test set.